src/Symfony/Component/Notifier/Bridge/Sendinblue/Tests/SendinblueTransportFactoryTest.php

```diff
@@ -22,32 +23,46 @@ public function testCreateWithDsn()
     {
         $factory = $this->createFactory();
 
-        $dsn = 'sendinblue://apiKey@default?sender=0611223344';
-        $transport = $factory->create(Dsn::fromString($dsn));
-        $transport->setHost('host.test');
+        $transport = $factory->create(Dsn::fromString('sendinblue://apiKey@host.test?sender=0611223344'));
 
         $this->assertSame('sendinblue://host.test?sender=0611223344', (string) $transport);
     }
 
-    public function testCreateWithNoPhoneThrowsMalformed()
+    public function testCreateWithMissingOptionSenderThrowsIncompleteDsnException()
     {
         $factory = $this->createFactory();
 
         $this->expectException(IncompleteDsnException::class);
 
-        $dsnIncomplete = 'sendinblue://apiKey@default';
+        $factory->create(Dsn::fromString('sendinblue://apiKey@host.test'));
+    }
+
+    public function testCreateWithNoApiKeyThrowsIncompleteDsnException()
+    {
+        $factory = $this->createFactory();
+
+        $this->expectException(IncompleteDsnException::class);
+
+        $dsnIncomplete = 'sendinblue://@default';
```

Comment:
both key and sender are missing here, we don't know which one triggers the exception.

Reply:
But there is only a token, password is not used 🤔

Reply:
I'm talking about the `sender` querystring parameter that also triggers a `IncompleteDsnException`